media/js/wiki-samples.js

```diff
@@ -0,0 +1,90 @@
+(function(win, doc, $) {
+    if(!win.waffle || !win.waffle.flag_is_active('wiki_samples')) return;
+
+    var sites = ['jsfiddle', 'codepen'];
+    var frameLength = 'frame_'.length;
+
+    // using id to get sample code section since macros discard other attributes
+    $('.sample-code-frame').after(function() {
+        var section = $(this).attr('id').substring(frameLength);
+        return createSampleButtons(section, sites);
+    });
+    $('.sample-code-table').after(function(){
+        var section = $(this).find('iframe').attr('id').substring(frameLength);
+        return createSampleButtons(section, sites);
+    });
```

Comment:
I may be missing something, but can this be condensed to the following? It uses lines 8-11 with a different selector.

```javascript
$('.sample-code-frame, .sample-code-table frame').after(function() {
    var section = $(this).attr('id').substring(frameLength);
    return createSampleButtons(section, sites);
});
```